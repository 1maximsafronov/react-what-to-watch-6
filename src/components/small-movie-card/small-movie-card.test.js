import React from "react";
import renderer from "react-test-renderer";
import SmallMovieCard from "./small-movie-card.jsx";

const mockMovie = {
  name: `movie name`,
  previewImage: `movie image`,
};

it(`Карточка фильма в каталоге отрисовывается верно`, () =>{
  const tree = renderer.create(
      <SmallMovieCard
        movie={mockMovie}
        onHover={() => {}}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
