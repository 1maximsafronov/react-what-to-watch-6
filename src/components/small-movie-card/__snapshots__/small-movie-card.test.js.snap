// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Карточка фильма в каталоге отрисовывается верно 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <img
      alt="movie name"
      height="175"
      src="movie image"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      movie name
    </a>
  </h3>
</article>
`;
